README.md

```diff
@@ -169,3 +169,90 @@ ID	Name		UUID
 1	Test-1		dc329f87d4de47198cfd2e21c6105b01
 2	Test-2		dc229f87d4de47198cfd2e21c6105b01
 ```
+
+Example (Connect to libvirt via TLS over TCP)
+-------
+
+```go
+package main
+
+import (
+        "crypto/tls"
+        "crypto/x509"
+
+        "fmt"
+        "io/ioutil"
+        "log"
+
+        "github.com/digitalocean/go-libvirt"
+)
+
+func main() {
+        // This dials libvirt on the local machine
+        // It connects to libvirt via TLS over TCP
+        // To connect to a remote machine, you need to have the ca/cert/key of it.
+        keyfilexml, err := ioutil.ReadFile("/etc/pki/libvirt/private/clientkey.pem")
+        if err != nil {
+                log.Fatalf("%v", err)
+        }
+
+        certfilexml, err := ioutil.ReadFile("/etc/pki/libvirt/clientcert.pem")
+        if err != nil {
+                log.Fatalf("%v", err)
+        }
+
+        cafilexml, err := ioutil.ReadFile("/etc/pki/CA/cacert.pem")
+        if err != nil {
+                log.Fatalf("%v", err)
+        }
+        cert, err := tls.X509KeyPair([]byte(certfilexml), []byte(keyfilexml))
+        if err != nil {
+                log.Fatalf("%v", err)
+        }
+
+        roots := x509.NewCertPool()
+        roots.AppendCertsFromPEM([]byte(cafilexml))
+
+        config := &tls.Config{
+                Certificates: []tls.Certificate{cert},
+                RootCAs:      roots,
+        }
+
+        // Use host name or IP which is valid in certificate
+        c, err := tls.Dial("tcp", "10.10.10.10", config)
+        if err != nil {
+                log.Fatalf("failed to dial libvirt: %v", err)
+        }
+
+        // Drop a byte before libvirt.New(c)
+        // More details at https://github.com/digitalocean/go-libvirt/issues/89
+        // Remove this line if the issue does not exist any more
+        c.Read(make([]byte, 1))
+
+        l := libvirt.New(c)
+        if err := l.Connect(); err != nil {
+                log.Fatalf("failed to connect: %v", err)
+        }
+
+        v, err := l.Version()
+        if err != nil {
+                log.Fatalf("failed to retrieve libvirt version: %v", err)
+        }
+        fmt.Println("Version:", v)
+
+        domains, err := l.Domains()
```

Comment:
```suggestion
        // Return both running and stopped VMs
        flags := libvirt.ConnectListDomainsActive | libvirt.ConnectListDomainsInactive
        domains, _, err := l.ConnectListAllDomains(1, flags)
```
Because `Domains()` is deprecated.